CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.109]
+### Changed
```

Comment:
Maybe prefix this as `Fixed` instead of `Changed`.

Reply:
Addressed.